tests/test_pipreqs.py

```diff
@@ -203,7 +219,7 @@ def test_ignored_directory(self):
         Test --ignore parameter
         """
         pipreqs.init(
-            {'<path>': self.project_with_ignore_directory, '--savepath': None, 
+            {'<path>': self.project_with_ignore_directory, '--savepath': None,
```

Comment:
this changes (probably made by your IDE) only muddles the commit, could you remove them?